FLIR/conservator/connection.py

```diff
@@ -31,8 +31,6 @@ class ConservatorMalformedQueryException(Exception):
     fault.
     """
 
-    pass
-
```

Comment:
The class definition is empty, shouldn't the pass statement be left here?

Reply:
PyLint seemed to think it was unnecessary, but I'll put both pass statements back in.